themes/custom/droopler_theme/scss/components/_geysir_overrides.scss

```diff
@@ -191,8 +191,10 @@ $geysir-add-more-button: $color-hecuba !default;
 
   // Button.
   .button {
-    border: none;
-    background: transparent;
+    &:not(.media-library-item__remove) {
```

Comment:
Nie idzie tego jakos inaczej złapać?
Boję się, że potem się pojawią jakieś przypadki gdzie jednak będzie jeszcze jakiś inny przycisk od czegoś, gdzie znów trzeba było by robić wykluczenia

Reply:
@mariusz-andrzejewski-droptica @grzegorz-pietrzak-droptica Nie mam pomysłu jak to inaczej zrobić. Można ewentualnie zrobić w drugą stronę, że ustawiamy to nie na wszystkich .button tylko na tych co chcemy tylko pytanie ile tego będzie.

Reply:
Spoko zostawiamy